js-exercises/minima/minima.js

```diff
@@ -0,0 +1,28 @@
+const isNumber = (num) => typeof num === 'number' && !Number.isNaN(num);
+
+function minima(numberOfElementsToReturn, array) {
+  const isValidNumber = isNumber(numberOfElementsToReturn);
+
+  if (!isValidNumber) {
+    throw new TypeError(`expected number , got ${typeof numberOfElementsToReturn}`);
+  }
+
+  if (numberOfElementsToReturn < 0) {
+    throw new Error('length must be greater or equal to zero.');
+  }
+
+  if (!Array.isArray(array)) {
+    throw new TypeError(`expected array , got ${typeof array}`);
+  }
+
+  const inputArray = array.filter(element => isNumber(element));
+  inputArray.sort((firstNum, secondNum) => firstNum - secondNum);
```

Comment:
```suggestion
  inputArray.sort((num1, num2) => num1 - num2);
```
Your names are good. You don't need very verbose names everywhere. Here's a simple thumb rule -- 
The bigger the scope of a variable, the more verbose the name should be. Since the numbers are scoped to a single line, you can use a shorter variable name.

Reply:
Also, you can name the `inputArray` to `sortedArray`